api/indexer.oas2.json

```diff
@@ -455,9 +651,6 @@
           {
             "$ref": "#/parameters/next"
           },
-          {
```

Comment:
was this also never implemented?

Reply:
Yes, while looking at which API endpoints supported the "Round" parameter, I noticed that params.Round was never used in the [implementation of /v2/assets/{asset-id}/balances](https://github.com/algorand/indexer/blob/develop/api/handlers.go#L448-L486) but listed as a parameter, so I got rid of it. CC @winder

Reply:
Thanks. This closes #374 

cc: @algoanne